stix_shifter_utils/utils/error_response.py

```diff
@@ -123,7 +123,10 @@ def call_module_error_mapper(json_data, return_object, connector ):
         if 'stix_translation.py' == caller_path_list[-1] or 'base_entry_point.py' == caller_path_list[-1]:
             module_path = 'stix_shifter_utils.stix_translation.stix_translation_error_mapper'
         else:
-            caller_module_name = caller_path_list[-3:-1]
+            if connector == "azure_sentinel":
```

Comment:
why do you need this error handling in utility?

Reply:
it is required as i am creating different folders for the different apis. This error handling will not be required after making the changes you suggested, will remove it in the next push